simpleclient/src/main/java/io/prometheus/client/Summary.java

```diff
@@ -1,18 +1,16 @@
 package io.prometheus.client;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Please list all the imports explicitly so it's easy to figure out where they're coming from